src/lib/createElement.js

```diff
@@ -1,5 +1,40 @@
 import { addEvent } from "./eventManager";
 
-export function createElement(vNode) {}
+export function createElement(vNode) {
+  if (vNode == null || typeof vNode === "boolean") {
+    return document.createTextNode("");
+  }
 
-function updateAttributes($el, props) {}
+  if (typeof vNode === "string" || typeof vNode === "number") {
+    return document.createTextNode(vNode);
+  }
+
+  if (Array.isArray(vNode)) {
+    const docFrag = document.createDocumentFragment();
+    docFrag.append(...vNode.map(createElement));
+    return docFrag;
+  }
+
+  const el = document.createElement(vNode.type);
+  vNode.children.map(createElement).forEach((child) => el.appendChild(child));
```

Comment:
한 가지 작은 부분인데, 위 부분에서 map과 forEach를 혼용하는 대신 forEach로 바로 순회하면서 appendChild를 호출해보면 어떨까요?
예를 들면,

```suggestion
vNode.children.forEach(child => el.appendChild(createElement(child)));
```

불필요하게 배열을 생성하지 않아서 성능을 조금 더 개선할 수 있을 것 같아요!